### ISSUE TITLE
Data Science with Python : Normalization & Standardization

### ISSUE THREAD

**Issue by prathimacode-hub** [girlscript/winter-of-contributing#471]
Welcome to 'DSWP' Team, good to see you here

This issue will helps readers in giving all the guidance that one needs to learn about Normalization & Standardization. Tutorial to Normalization & Standardization and how it's applied using sample code.

To get assigned to this issue, add your serial numbers mentioned in the spreadsheet of "Data Science with Python", the approach one would follow and choice you prefer (Documentation, Audio, Video). You can go with all three or any number of options you're interested to work on.

If you had referred any resources, add them up in "DS Resources". Similarly if you had used datasets, include them in "DS Datasets".

Domain : Machine Learning

Mentors Assigned : [Dhruv Bajaj](https://github.com/DhruvBajaj01) / Jivitesh Jain

Points to Note :

The issues will be assigned on a first come first serve basis, 1 Issue == 1 PR.

"Issue Title" and "PR Title should be the same. Include issue number along with it.

Changes should be made inside the Datascience_With_Python/ directory & Datascience_With_Python branch.

Follow Contributing Guidelines & Code of Conduct before start Contributing.

This issue is only for 'GWOC' contributors of 'DSWP' domain.

All the best. Enjoy your open source journey ahead. 😎

**Comment by pankaj-Makhijani** [comment]
I would like to work on this.
Pankaj Makhijani - 432
Issue No - #471 
Type: Doccumentation
Approach: 
1 Feature Scaling?
2 What is Normalization?
3 What is Standardization?
4 Implementing Feature Scaling in Python
   a. Normalization using Sklearn library
   b. Standardization using Sklearn library

**Comment by prathimacode-hub** [comment]
Issue assigned to @pankaj-Makhijani for documentation.

**Comment by Samhitha-Jasti** [comment]
I would like to work on this.
Anjani Samhitha -209
Batch - 6
Issue No - #471
Type: Doccumentation
Approach:
Introduction to Normalization, Types of Normalization, Uses and Coding tutorial

**Comment by jiviteshjain** [comment]
Hey @Samhitha-Jasti, this issue is already assigned to @pankaj-Makhijani for documentation. If you feel there is something here you can cover in an audio/video tutorial, let us know!

**Comment by kunal-kumar-chaudhary** [comment]
I can contribute to this through video. 
kunal kumar - 582
batch -16 
issue no. - #471
type - video
approach - I will explain the concepts first regarding why feature scaling is needed, what are different ways in which we can perform feature scaling, and then will implement it in python. How does graphs looks like before and after feature scaling if we try and plot the independent variable with respect to the dependent variable. it would further enhance the understanding of any newcomer.

**Comment by Samhitha-Jasti** [comment]
@jiviteshjain Yes I can take the audio contribution on the topic "Introduction to Normalization, Types of Normalization, Uses and Coding tutorial"

**Comment by prathimacode-hub** [comment]
Issue assigned to @Samhitha-Jasti for audio contribution

Issue assigned to @kunal-kumar-chaudhary for video contribution

**Comment by Samhitha-Jasti** [comment]
@jiviteshjain Sir i made a pull request of mp3 file on Normalization and Standardization, can you please review it

**Comment by pankaj-Makhijani** [comment]
@jiviteshjain Hello sir I have prepared the documentation, can you please review it.

**Comment by pankaj-Makhijani** [comment]
@prathimacode-hub can I unassign myself from this issue as my PR is merged? I am not able to contribute to other issues as this issue is still open.
